lua/nvim-treesitter/install.lua

```diff
@@ -107,7 +106,11 @@ local function get_command(cmd)
     end
   end
 
-  return string.format('%s%s', ret, options)
+  local final = string.format('%s%s', ret, options)
+  if cmd.opts and cmd.opts.cwd then
+    final = final .. shell.select_temp_directory_change_command_end()
+  end
+  return final
```

Comment:
It would be probably better to just have one function to set the temporary directory that can be called at the end of this function to surround the actual command.

Reply:
yeah you are right :) it will be nicer :)